CHANGELOG.md

```diff
@@ -13,7 +15,7 @@
 
 ### Fixes
 
-- Send DidBecomeActiveNotification when OOM enabled (#905)
+* Send DidBecomeActiveNotification when OOM enabled (#905)
```

Comment:
The auto-updater uses `-`, that should probably be aligned?

Reply:
Most likely, good catch, auto formatter changed automatically for me.

Reply:
https://github.com/getsentry/sentry-dart/issues/922